client/src/app/game/soundButton.module.css

```diff
@@ -0,0 +1,8 @@
+.soundButton {
+	position: absolute;
```

Comment:
Chat uses position fixed. It shouldn't matter since we have no scrolling but we should keep it consistent.